libs/sdk/src/common/push-protocol.ts

```diff
@@ -9,6 +9,23 @@ import AWF_Config from './config';
 import Web3Connector from './web3.connector';
 import { PushStream } from '@pushprotocol/restapi/src/lib/pushstream/PushStream';
 
+interface PushOrgNotification {
+  cta: string;
+  title: string;
+  message: string;
+  icon: string;
+  url: string;
+  sid: string;
+  app: string;
+  image: string;
+  blockchain: string;
+  notification: {
+    body: string;
+    title: string;
+  };
+  secret: string;
```

Comment:
all the props are mandatory?

Reply:
couldn't this be used as a ReturnType from the push sdk?

Reply:
The return type they have in declaration file of sdk is `any`, so I created an interface for it manually.